source/includes/baremetal/sshkey.md

```diff
@@ -0,0 +1,121 @@
+## SSH Keys
+
+Allows you to create, delete, and manage your SSH keys.
+
+Create and manage your SSH keys.
```

Comment:
Just one of these descriptions is enough

Reply:
Resolved